lldb/packages/Python/lldbsuite/test/decorators.py

```diff
@@ -886,11 +885,14 @@ def skipIfAsan(func):
     """Skip this test if the environment is set up to run LLDB *itself* under ASAN."""
     return skipTestIfFn(is_running_under_asan)(func)
 
-def _get_bool_config_skip_if_decorator(key):
```

Comment:
Should this part be upstream?

Reply:
Sounds good to me. I think we anyway need to wait with this until we tracked down the regressions, I'll upstream that and then rebase this on top.

Reply:
Thanks, it's more about making sure it doesn't get forgotten than blocking the merge.